packages/sdk/README.md

```diff
@@ -1,39 +1,174 @@
-# Apillon SDK for NodeJS
+# Apillon SDK
 
 [![npm version](https://badge.fury.io/js/@apillon%2Fsdk.svg)](https://badge.fury.io/js/@apillon%2Fsdk)
 [![Twitter Follow](https://img.shields.io/twitter/follow/Apillon?style=social)](https://twitter.com/intent/follow?screen_name=Apillon)
 
-Apillon SDK is a NodeJS library written in Typescript for using Apillon Web3 services.
+Libraries and tools for interacting with your Apillon integration.
+Apillon SDK reduces the amount of work required to use Apillons REST API. It reduces the boilerplate code you need to write
+as well as compresses multi step flows into single operations.
 
 ## Requirements
 
-To be able to use Apillon CLI, you must register an account at [Apillon.io](https://app.apillon.io), create a project and generate an API key with appropriate permissions.
+- npm 8.4.0 or higher
+- node.js 16.17.0 or higher
+- Apillon API key and secret
 
-## Installation
+## Getting started
 
-To install the SDK, run the following command:
+SDK is available on [NPM](https://www.npmjs.com/package/@apillon/sdk) and you can also check it out directly on [GitHub](https://github.com/Apillon/sdk).
+
+### Installation
 
 ```sh
 npm install @apillon/sdk
 ```
 
-## Usage
+### Initialization
 
-To use the SDK in your project, first import appropriate module:
+```ts
+import { Hosting } from "@apillon/sdk";
 
-```typescript
-import { Hosting } from '@apillon/sdk';
+const hosting = new Hosting({
+  apiKey: "",
+  apiSecret: "",
+});
 ```
 
-Then, create a new instance of the module class:
+Apillon SDK consists of different modules depending on which service you want to use. But all modules require the same initial config of `apiKey` and `apiSecret` shown above in `Hosting` module example.
 
-```typescript
-const hosting = new Hosting({
-    key: 'yourApiKey',
-    secret:'yourApiSecret'
+## Hosting
+
+Hosting module encapsulates functionalities for Hosting service available on Apillon dashboard.
+
+::: tip
+You can only create a new webpage through the [dashboard hosting service](https://app.apillon.io/dashboard/service/hosting).
+:::
+
+The flow of deploying a new website looks like this:
+
+1. upload new website files
+2. trigger deploy to staging
+3. trigger deploy from staging to production
+
+You can also directly deploy uploaded files to production.
+
+### Usage example
+
+```ts
+import { Hosting } from "@apillon/sdk";
```

Comment:
Example should be working out of the box. This one is not. That's why I suggest that we move examples to separate project inside this monorepo. But for now just fix all the examples!